api/tests/test_manage_asset_api.py

```diff
@@ -121,6 +126,24 @@ def test_admin_can_post_asset(self, mock_verify_id_token):
         self.assertEqual(Asset.objects.count(), count + 1)
         self.assertEqual(response.status_code, 201)
 
+    @patch("api.authentication.auth.verify_id_token")
+    def test_non_super_admin_cannot_patch_invoice_receipt(self, mock_verify_id_token):
```

Comment:
Can a super_admin patch invoice receipt?

Reply:
Yes, a super_admin can patch an invoice receipt.

Reply:
Could you add test to show that super admin can patch invoice receipt if you've not already done that.